langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -51,10 +51,10 @@ public type Comment xml;
 @builtinSubtype
 public type Text xml;
 
-# Returns number of xml items in `x`.
+# Returns number of xml items in an xml value.
```

Comment:
```suggestion
# Returns number of XML items in an XML value.
```
or
```suggestion
# Returns number of XML items in an `xml` value.
```

Maybe we can discuss and fix this separately? Same applies for other comments re: XML lower/upper case.